File: semester_1/c/exp6/4.c
#include <stdio.h>

int ISPRIME(int a ){
    for(int i = 2  ; i < a ; i++){
        if (a%i == 0){
            return -1;
        }
    }
    return 1;
}

int main(){
    int start;
    int end;
    printf("enter the starting range: ");
    scanf("%d",&start);
    printf("enter the ending range: ");
    scanf("%d",&end);
    for (int i = start ; i <=end ; i ++){
        if(ISPRIME(i) == -1){
            printf("%d is not prime\n",i);
        }
        else{
            printf("%d is prime\n",i);
        }
    }
    return 0;
}

File: semester_2/python/experiments/4/4.py
n: int = int(input("Enter the number "))
isPrime: bool = True
for i in range(2,int(n**0.5) + 1):
    if n % i == 0:
        isPrime = False
        break

if n <= 1:
    isPrime = False

if isPrime:
    print("Number is Prime")
else:
    print("Number is not Prime")
